tests/integration/raftlog.py

```diff
@@ -32,11 +33,14 @@ def from_file(cls, _file):
         elements = int(mb_line[1:])
 
         args = []
+        locations = []
```

Comment:
This is really good!

Reply:
note, its just the "data" part of the multiblk element, not the start of the element. so can only overwrite up to length of element.